src/FakeItEasy/Configuration/ICallOrderAssertConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+namespace FakeItEasy.Configuration
+{
+    /// <summary>
+    /// Allows the developer to check that an asserted call happened in order.
+    /// </summary>
+    public interface ICallOrderAssertConfiguration

```

Comment:
When I read the name I didn't like it too. It felt a little better once I saw it in context. But if we're looking for inspiration in the code, we could follow `IAfterCallSpecifiedConfiguration` and try `IAfterMustHaveHappenedConfiguration`…